src/Form/Control.php

```diff
@@ -100,7 +100,6 @@ protected function init(): void
     public function set($value = null, $junk = null)
     {
         if ($this->field) {
-            $value = $this->getApp()->ui_persistence->typecastLoadField($this->field, $value);
```

Comment:
This was wrong here. `Control::set` should expect already fully loaded data and can not be used to set data from frontend directly.